src/style.css

```diff
@@ -1 +1,94 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  cursor: pointer;
```

Comment:
set `cursor: pinter` for elements you really need
it's not a good practice to set it for all elements